src/components/MultiHopTrade/components/TradeAmountInput.tsx

```diff
@@ -178,6 +220,66 @@ export const TradeAmountInput: React.FC<TradeAmountInputProps> = memo(
       )
     }, [assetSymbol, cryptoAmount, fiatAmount, isFiat])
 
+    const renderController: RenderController = useCallback(
+      ({ field: { onChange } }) => {
+        return (
+          <NumberFormat
+            customInput={AmountInput}
+            isNumericString={true}
+            disabled={isReadOnly}
+            _disabled={numberFormatDisabled}
+            suffix={isFiat ? localeParts.postfix : ''}
+            prefix={isFiat ? localeParts.prefix : ''}
+            decimalSeparator={localeParts.decimal}
+            inputMode='decimal'
+            thousandSeparator={localeParts.group}
+            value={isFiat ? bnOrZero(fiatAmount).toFixed(2) : formattedCryptoAmount}
+            // this is already within a useCallback, we don't need to memo this
+            // eslint-disable-next-line react-memo/require-usememo
+            onValueChange={(values: NumberFormatValues) => {
+              // Controller onChange
+              onChange(values.value)
+              handleValueChange(values)
+
+              const value = values.value
+              if (isFiat) {
```

Comment:
This looks like a scary breaking change but it is not - this sets the value of the newly introduced field context, which we don't consume anywhere here or in current cosumers, only in RFOX - in fact RFOX consumption is the only time where a form context would be existing here, else this sets the dummy context created above, it being dummy proving this isn't consumed anywhere